test/components/creeps/creepActions.test.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="../../../typings/globals/screeps/index.d.ts"/>
```

Comment:
Unfortunately I couldn't figure out how to do it. 

I had to add this reference path to be able to reference the global screeps typings inside the tests. 

Maybe you can take a look at it?